xds/internal/balancer/clustermanager/clustermanager_test.go

```diff
@@ -732,10 +732,10 @@ func (s) TestUpdateStatePauses(t *testing.T) {
 	bal := builder.Build(cc, balancer.BuildOptions{})
 
 	configJSON1 := `{
-"children": {
-	"cds:cluster_1":{ "childPolicy": [{"update_state_balancer":""}] }
-}
-}`
+ "children": {
+	 "cds:cluster_1":{ "childPolicy": [{"update_state_balancer":""}] }
+ }
+ }`
```

Comment:
It seems this whole file can be reverted?

Reply:
Yeah. I don't remember what caused these changes. I think `gofmt` might be the culprit. Reverted anyways.